content/influxdb/cloud-dedicated/get-started/write.md

```diff
@@ -157,27 +157,31 @@ request to the InfluxDB API `/api/v2/write` endpoint using the `POST` request me
 Include the following with your request:
 
 - **Headers**:
-  - **Authorization**: Token <INFLUX_TOKEN>
+  - **Authorization**: Bearer <INFLUX_TOKEN>
   - **Content-Type**: text/plain; charset=utf-8
   - **Accept**: application/json
 - **Query parameters**:
-  - **org**: _This parameter is ignored and can be an arbitrary string_
   - **bucket**: InfluxDB database name
   - **precision**: Timestamp precision (default is `ns`)
 - **Request body**: Line protocol as plain text
 
+{{% note %}}
+With the InfluxDB Cloud Dedicated v2 API `/api/v2/write` endpoint, `Authorization: Bearer` and `Authorization: Token` are equivalent and you can use either scheme to pass a database token in your request. For more information about HTTP API token schemes, see how to [authenticate API requests](/influxdb/cloud-dedicated/primers/api/v2/#authenticate-api-requests).
+{{% /note %}}
+
 The following example uses cURL and the InfluxDB v2 API to write line protocol
 to InfluxDB Cloud dedicated:
 
 {{% influxdb/custom-timestamps %}}
 ```sh
 export INFLUX_HOST=https://cluster-id.influxdb.io
-export INFLUX_ORG=ignored
 export INFLUX_TOKEN=DATABASE_TOKEN
 
+DATABASE_NAME=get-started
```

Comment:
Any reason why you export environment variables for the others, but create a shell variable for the database?

Reply:
I guess for this example, none of them need to be exported.